Makefile

```diff
@@ -0,0 +1,32 @@
+BINPATH ?= build
+
+BUILD_TIME=$(shell date +%s)
+GIT_COMMIT=$(shell git rev-parse HEAD)
+VERSION ?= $(shell git tag --points-at HEAD | grep ^v | head -n 1)
+
+LDFLAGS = -ldflags "-X main.BuildTime=$(BUILD_TIME) -X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)"
+
+.PHONY: all
+all: audit test build
+
+.PHONY: audit
+audit:
+	nancy go.sum
```

Comment:
Update to:

`go list -m all | nancy sleuth`

Reply:
Actually, it needs to be `go list -json -m all | nancy sleuth` now as Mariana spotted that the advice from nancy has been updated again.

Reply:
👍  good call @jessjenkins

Reply:
done